### PR TITLE
Add uuid regex

### PR DESCRIPTION
**Pull request by porksmash**

### CODE REVIEW

File: smarturls.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "int4": "\d{4,4}",
     "slug": "[\w-]+",
     "username": "[\w.@+-]+",
+    "uuid": "[0-9a-f]{32}",
```
Comment (amitu) [inline smarturls.py]: We should also support UUIDs like: de305d54-75b4-431b-adb2-eb6b9e546014, according to wikipedia this is canonical format.

**Comment by porksmash** [comment]
Just updated with a regex that matches canonical format and without dashes.

**Comment by amitu** [comment]
Thank you :-)

Will create a new release soon.

**Comment by amitu** [comment]
0.1.6 release is out, thanks @porksmash :-). It also contains base64, and ekey formats.

**Pull request merged**